src/content/docs/accounts/accounts-billing/account-setup/downgradecancel-account.mdx

```diff
@@ -57,21 +57,49 @@ If you have a Pro or Enterprise edition with [a long-term contract](/docs/accoun
   </Collapser>
 </CollapserGroup>
 
-## Delete organization or accounts [#cancel]
+## Cancel an account [#cancel-account]
 
-If you're seeking to delete one or more accounts in an organization, but keep your organization, contact [support](/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal).
+If you're using the Pro or Enterprise edition, or if you've downgraded to the Free edition, you can cancel one or more accounts while keeping your organization.
 
-If you're seeking to delete your New Relic organization entirely, including any and all accounts within it, your organization must meet the following requirements:
+When you cancel an account:
 
-* It has only one account, and one user.
-  * If you have more than one account in your organization, contact [support](/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal).
-  * If you have more than one user, you can delete users via the [User management UI](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks/#delete-users) or via [Nerdgraph](/docs/apis/nerdgraph/examples/nerdgraph-manage-users/#delete-users).
-* It's on our newer pricing model. 
-  * Most organizations are, but if you're not sure, see [Pricing models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models/#determine-pricing) for how to determine that. If you're on the older pricing model, contact [support](/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal).
-* It must not have an active paid subscription.
-  * If you have an active paid subscription, see the section [Downgrade your edition](#downgrading) above in this document.
+* It is disabled, but it's still present in your organization in **Accounts > Cancelled accounts**.
+* Users, groups, and custom roles remain, but access grants that are assigned to that cancelled account lose all connections to the account. You should also remove any access grant associated with this account, otherwise users may see errors when they try to view it.
+* Any API keys associated with a deleted account become inactive and can't be reassigned to another account. 
 
-### To delete your organization, follow these steps:
+### Who can cancel an account [#who-cancel]
+
+To cancel an account you must be a user who is:
+
+* A full platform user.
+* In a group that has the Organization [administrative setting](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts/#admin-settings).
+
+### How to cancel an account [#how-to-cancel]
+
+You can only cancel an account using our NerdGraph API&mdash;not through the UI.
```

Comment:
the emdash is a weird choice to me. I'd make it too separate sentences just to make it crystal clear:

You can only cancel an account via our NerdGraph API. You can't cancel an account in the UI.

Reply:
@paperclypse I removed the em-dash and added a second sentence.